### ISSUE TITLE
Attempting to connect without a "Real Name" fails

### ISSUE THREAD

**Issue by garetmckinley** [KokoIRC/koko#33]
Not sure if this is a limitation of IRC or if this is a koko-specific issue, but when I attempt to connect without entering a "Real Name", it just puts me in a attempting-to-connect loop.

![screen shot 2016-11-10 at 10 22 42 am](https://cloud.githubusercontent.com/assets/1624279/20182390/a14a959c-a72f-11e6-93f6-5bb833901391.png)

And it just keeps doing that until I close the window and reconnect with a real name.

**Comment by WeeSkilz** [comment]
I just had this issue.

For any blank name-based fields, the nickname could just be aliased if the field isn't populated?

**Comment by garetmckinley** [comment]
I agree, maybe we could make the placeholder text mirror the nick to imply to the user that it will be auto populated.

Or maybe it would be better to obscure some fields under a hidden "advanced" panel. Ill put some thought into it
